repositories.yaml

```diff
@@ -968,16 +968,15 @@ applications:
     canonical_app_name: Mozilla VPN Cirrus Sidecar
     app_description: |
       The sidecar Cirrus container for the Mozilla VPN server
-    url: https://github.com/mozilla/experimenter
+    url: https://github.com/mozilla-services/guardian-website
```

Comment:
That repo doesn't exist (or is not public). probe-scraper will fail on that. Should this be just https://github.com/mozilla-mobile/mozilla-vpn-client ?